src/NoteMouseHandler/PencilMouseHandler.js

```diff
@@ -0,0 +1,184 @@
+import _ from "lodash"
+import SharedService from "../services/SharedService"
+import MouseHandler, { defaultActionFactory, getLocal } from "./NoteMouseHandler"
+import { pointSub } from "../helpers/point"
+
+export default class PencilMouseHandler extends MouseHandler {
+  constructor(emitter) {
+    super(emitter, [defaultActionFactory, actionFactory], getCursor)
+  }
+}
+
+function actionFactory(e, ctx) {
+  const cpos = getLocal(e)
+  const rects = ctx.getEventsUnderPoint(cpos.x, cpos.y)
+
+  if (e.nativeEvent.button != 0) {
```

Comment:
__[jshint]__
*  Use '!==' to compare with '0'.


<!-- Comment by [SideCI](https://sideci.com) -->